src/kepler/tests.zig

```diff
@@ -0,0 +1,306 @@
+const std = @import("std");
+const os = @import("root").os;
+const kepler = os.kepler;
+
+fn rdtsc() u64 {
+    var eax: u32 = undefined;
+    var edx: u32 = undefined;
+    asm volatile ("rdtsc"
+        : [_] "={eax}" (eax),
+          [_] "={edx}" (edx)
```

Comment:
Put the comma on this line to be consistent with other inline `asm` in this project
```
: [first_output]
, [second_output]
// ...
```